components/server/ee/src/github/github-app-support.ts

```diff
@@ -53,12 +53,12 @@ export class GitHubAppSupport {
         const listReposForInstallation = async (installation: RestEndpointMethodTypes["apps"]["getUserInstallation"]["response"]) => {
             const sub = await probot.auth(installation.data.id);
             try {
-                const accessibleRepos = (await sub.paginate(sub.rest.apps.listReposAccessibleToInstallation, { per_page: 100 }));
+                const { repositories: accessibleRepos } = (await sub.paginate(sub.rest.apps.listReposAccessibleToInstallation, { per_page: 100 }));
```

Comment:
last time I checked, this was a mismatch between typings and the actual shape of the result, i.e. I had to change the other way around because the `sub.paginate` flattens the result. we should pay attention to that.

Reply:
Ok, makes sense. I will re-do and translate your explanation into a comment. Let's add this to the test issue.